assembly-multiuser/dashboard/src/main/patches/src/app/index.module.ts.patch

```diff
@@ -30,7 +30,7 @@
 +        new Promise((resolve, reject) => {
 +          const script = document.createElement('script');
 +          script.async = true;
-+          script.src = keycloakSettings['che.keycloak.auth-server-url'] + '/js/keycloak.js';
++          script.src = keycloakSettings['che.keycloak.auth_server_url'] + '/js/keycloak.js';
 +          script.addEventListener('load', resolve);
 +          script.addEventListener('error', () => reject('Error loading script.'));
```

Comment:
do we need to add on the error that it's about keycloak ? like
"Error loading keycloak script"

Reply:
Might be, but it's not really related to this PR...